### PR TITLE
Do not close dataplane if we update local-dst with local-dst on nsm request

### PR DESCRIPTION
**Pull request by denis-tingajkin**
## Description
https://github.com/networkservicemesh/networkservicemesh/issues/1222
## How Has This Been Tested?

- [X] Covered by existing integration testing
- [ ] Added integration testing to cover
- [ ] Tested locally
- [ ] Have not tested

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: dataplane/vppagent/pkg/vppagent/resync_manager.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	}
+	if conn.State != connection.State_DOWN {
+		return
+	}
+	m.storedDataChanges.Delete(conn.Id)
+}
+
+//TODO: do not use pointer to poinder
+func (m *resyncManager) needToResync(id string, dataChange **configurator.Config) bool {
+	if value, ok := m.storedDataChanges.Load(id); ok {
```
Comment (denis-tingajkin) [inline dataplane/vppagent/pkg/vppagent/resync_manager.go]: TODO: add check on namespace

File: controlplane/pkg/nsm/nsm.go [hunk 2]
```diff
@@ -276,15 +276,6 @@ func (srv *networkServiceManager) request(ctx context.Context, request networkse
...
-	if existingCC != nil && existingCC.RemoteNsm == nil && cc.RemoteNsm == nil {
-		logrus.Errorf("NSM:(9.0-%v) Closing Dataplane because of existing connection passed...", requestID)
-		if err = srv.closeDataplane(existingCC); err != nil {
-			logrus.Errorf("NSM:(9.1-%v) Closing Dataplane error for local connection: %v", requestID, err)
-		}
-	}
-
 	var newXcon *crossconnect.CrossConnect
 
 	// 9.2 Sending updated request to dataplane.
```
Comment (lobkovilya) [inline controlplane/pkg/nsm/nsm.go]: Numeration should be fixed

File: .mk/code_analysis.mk [hunk 3]
```diff
@@ -4,11 +4,11 @@ lint-fix:
 
 .PHONY: lint-install
```
Comment (nickolaev) [inline .mk/code_analysis.mk]: Can these go in a separate PR please.

Comment (denis-tingajkin) [reply .mk/code_analysis.mk]: Yep, Let the build finish and I'll update this PR.

Comment (denis-tingajkin) [reply .mk/code_analysis.mk]: done

File: controlplane/pkg/nsm/nsm.go (hunk 2 above)
Comment (denis-tingajkin) [reply controlplane/pkg/nsm/nsm.go]: I guess fixed

File: dataplane/vppagent/pkg/vppagent/nsmonitor/monitor_netnsinode_server.go [hunk 4]
```diff
@@ -9,28 +9,32 @@ import (
...
 	"github.com/networkservicemesh/networkservicemesh/controlplane/pkg/apis/crossconnect"
 	"github.com/networkservicemesh/networkservicemesh/controlplane/pkg/apis/local/connection"
-	"github.com/networkservicemesh/networkservicemesh/controlplane/pkg/monitor"
 	monitor_crossconnect "github.com/networkservicemesh/networkservicemesh/controlplane/pkg/monitor/crossconnect"
 	"github.com/networkservicemesh/networkservicemesh/utils/fs"
 )
 
 type MonitorNetNsInodeServer struct {
-	monitor.Recipient
+	kvSchedulerClient   KVSchedulerClient
```
Comment (haiodo) [inline dataplane/vppagent/pkg/vppagent/nsmonitor/monitor_netnsinode_server.go]: Why we need this here?

Comment (haiodo) [reply dataplane/vppagent/pkg/vppagent/nsmonitor/monitor_netnsinode_server.go]: Ok, so it in dataplane now.
